order/fixed_points.lean

```diff
@@ -174,7 +174,7 @@ instance : complete_lattice (fixed_points f) :=
 { le           := λx y, x.1 ≤ y.1,
   le_refl      := λ x, le_refl x,
   le_trans     := λ x y z, le_trans,
-  le_antisymm  := λ x y hx hy, subtype.eq $ le_antisymm hx hy,
+  le_antisymm  := λ x y hx hy, .eq $ le_antisymm hx hy,
```

Comment:
I guess this doesn't compile?

Reply:
Huh, how did that end up happening? I don't recall ever opening that file...